### DISCUSSION TITLE
A dreamful Chinese student

### DISCUSSION THREAD

**Discussion by Chinesedreamer** [OptimalScale/LMFlow#115] [Ideas]
As a college student, I have many thoughts about entrepreneurship. The release of ChatGPT has deeply impressed me with the development of global technology. I understand that this technological revolution is relevant to everyone and represents an opportunity for ordinary people like us. My major is tourism management, and I have limited knowledge of computer-related subjects. I have some entrepreneurial ideas for ChatGPT, and although these ideas may seem naive to professionals, I hope that some genuinely enthusiastic technical experts would be willing to communicate with me and provide me with advice and guidance. Thank you all.

**Comment by Chinesedreamer** [comment]
Because it's my first time using GitHub, I'm not quite sure how to check messages here. If I'm not able to respond promptly, you can also contact me through my email at [tangjunyou672@gmail.com](mailto:tangjunyou672@gmail.com). Thank you all!

**Comment by shizhediao** [comment]
Hi,
Thank you for your interest! 

I am deeply touched by your entrepreneurial passion. You are welcome to join our community (see our README, there are several groups like WeChat), where you can connect with more people. 

We also look forward to hearing more of your ideas. Thank you!

**Reply by Chinesedreamer** [reply]
Thank you so much for getting back to me so quickly. To be honest, I'm so excited that I might not be able to sleep tonight. I'm not very experienced in this area and couldn't find your WeChat group or other contact information on GitHub. If it's convenient for you, could you please provide me with your contact information via email? I'm really looking forward to communicating with you. Thank you again for your enthusiasm and patience, and I wish you all the best with your project research!  my WeChat is dadajun770

**Comment by Chinesedreamer** [comment]
Thank you so much for getting back to me so quickly. To be honest, I'm so
excited that I might not be able to sleep tonight. I'm not very experienced
in this area and couldn't find your WeChat group or other contact
information on GitHub. If it's convenient for you, could you please provide
me with your contact information via email? I'm really looking forward to
communicating with you. Thank you again for your enthusiasm and patience,
and I wish you all the best with your project research!my WeChat is
dadajun770

shizhediao ***@***.***> 于2023年4月5日周三 02:34写道：

**Reply by shizhediao** [reply]
Hi, please take a look at https://lmflow.com/
